### ISSUE TITLE
A11y_AzureDataStudioSQL_Add Database reference_System Database_ScreenReader:Voice Over does not announces required when user navigates on Database name edit box with the help of tab key.

### ISSUE THREAD

**Issue by RohitJadhav-TCS** [microsoft/azuredatastudio#14522]
“[Check out Accessibility Insights! ](https://nam06.safelinks.protection.outlook.com/?url=https://accessibilityinsights.io/&data=02%7c01%7cv-stfe%40microsoft.com%7cb67b2c4b646d4f9561a208d6f4b5c39b%7c72f988bf86f141af91ab2d7cd011db47%7c1%7c0%7c636965458850501301&sdata=mxhokIKNMb22llsjXHLgU3XZibj1Qfx37rpY4PU2sfE%3D&reserved=0)- Identify accessibility bugs before check-in and make bug fixing faster and easier.”

**GitHubTags**: #A11yMAS;#A11yTCS;#A11ySev2;#SH_AzureDataStudioSQL-MAC-Feb2021;#SQL Azure Data Studio;#MAC;#FTP;#DesktopApp;#WCAG3.3.1;#VoiceOver;#AINotInScope;

### Environment Details:
Application Name: Azure Data Studio
Version: 1.26.1
Mac OS Version: Big Sur Version 11.2.2
Screen reader: Voice Over

### Repro Steps:

1. Launch the application-Azure Database Project and install extension SQL Database Projects.
2. Navigate to SQL Database Project Home Screen with heading "Projects".
3. Tab to connections select your server and connect to it.
4. Few databases will appear under your connection then right click on any database
5. Tab till "Create project from database" and hit enter.
6. New database project will get created under "Projects" tab and tab till "Database references"
7. Right Click and hit enter on "Add Database references".
8. Select "System Database" radio button.
9. Start the Voice Over,
10. Navigate to Database name edit box with Capslock + Right Arrow key or Tab and check the result.

### Actual:
Voice Over does not announces required when user navigates on Database name edit box with the help of tab key .

Similar issue is observed in below scenarios:
Add Database reference- Data tier application
Add Database reference- Projects

### Expected:
Voice Over should announce required when user navigates on Database name edit box with the help of tab key .

### User Impact:
Screenreader user will not able to know that it is a required field.

**Comment by Swati1700** [comment]
Verified the bug in the environment
Application Name: Azure Data Studio
macOS: Ventura 13.2
Issue is fixed and does not reproduced. Please find the not repro attachment below.
[Screen Recording (3).webm](https://user-images.githubusercontent.com/86710888/220585244-e41c86d1-8614-45a4-b16f-a8ff43632284.webm)
Github Tags:#Closed;
